### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getTabLength' of undefined

### ISSUE THREAD

**Issue by suda** [jamesmacfie/space-tab#17]
1. Trying to convert document containing both spaces and tabs to spaces

**Atom Version**: 0.207.0
**System**: Mac OS X 10.10.3
**Thrown From**: [space-tab](https://github.com/jamesmacfie/space-tab) package, v0.4.1

### Stack Trace

Uncaught TypeError: Cannot read property 'getTabLength' of undefined

```
At /Users/suda/.atom/packages/space-tab/lib/space-tab.coffee:27

TypeError: Cannot read property 'getTabLength' of undefined
  at Object.module.exports.convertToSpaces (/Users/suda/.atom/packages/space-tab/lib/space-tab.coffee:27:22)
  at atom-workspace.<anonymous> (/Users/suda/.atom/packages/space-tab/lib/space-tab.coffee:4:74)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-div.atom.commands.add.core:confirm (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:109:19)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:520:16)
  at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:355:22)
  at HTMLDocument.module.exports.WindowEventHandler.onKeydown (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:177:20)

```

### Commands

```
     -2:28 editor:indent (atom-text-editor.editor.is-focused)
     -2:27.8.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -2:27.8.0 snippets:expand (atom-text-editor.editor.is-focused)
     -2:27.8.0 editor:indent (atom-text-editor.editor.is-focused)
     -2:27.4.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -2:27.4.0 snippets:expand (atom-text-editor.editor.is-focused)
     -2:27.4.0 editor:indent (atom-text-editor.editor.is-focused)
  2x -2:23 core:move-down (atom-text-editor.editor.is-focused.autocomplete-active)
     -2:22.7.0 autocomplete-plus:confirm (atom-text-editor.editor.is-focused.autocomplete-active)
     -2:22.3.0 core:move-right (atom-text-editor.editor.is-focused)
     -2:21.1.0 core:save (atom-text-editor.editor.is-focused)
  7x -2:17.3.0 core:close (ul.list-inline.tab-bar.inset-panel)
     -0:05.6.0 command-palette:toggle (atom-text-editor.editor.is-focused)
     -0:02.2.0 core:move-down (atom-text-editor.editor.mini.is-focused)
     -0:02.0 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:02.0 space-tab:convert-to-spaces (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "destroyEmptyPanes": false,
    "disabledPackages": [
      "package-that-throws-an-exception",
      "package-with-broken-package-json",
      "package-with-broken-keymap",
      "symbols-view",
      "yuno-commit",
      "toolbar-basic",
      "linter",
      "regex-railroad-diagram",
      "refactor",
      "coffee-refactor",
      "compare-files"
    ],
    "themes": [
      "atom-dark-ui",
      "monokai"
    ]
  }
}
```

### Installed Packages

```coffee
# User
atom-ctags, v4.1.2
atom-django, v0.3.0
autocomplete-python, v0.4.2
coffee-navigator, v0.0.17
color-picker, v2.0.4
django-templates, v0.5.0
emmet, v2.3.10
file-type-icons, v0.6.0
indentation-indicator, v0.5.1
language-arduino, v0.2.1
language-batch, v0.3.3
language-ejs, v0.1.0
language-spark, v0.3.1
linter-eslint, v0.5.8
monokai, v0.14.0
pretty-json, v0.3.3
project-manager, v1.15.10
space-tab, v0.4.1
spark-dev, v0.0.23
spark-dev-cloud-functions, v0.0.5
spark-dev-cloud-variables, v0.0.3
symbols-tree-view, v0.9.3
tabs-to-spaces, v0.11.0
tool-bar, v0.1.6
tool-bar-main, v0.0.8
white-cursor, v0.6.0

# Dev
No dev packages
```

**Comment by jamesmacfie** [comment]
Fixed in 0.4.2

**Comment by rugk** [comment]
This happens again if you try the command with no files open. BTW I only tried this because a background tip gave me the info that this would highlight the tree view. (But when I tested it with the package disabled it folded/unfolded the current function or whatever - but anyway maybe the chosen keyboard shortcut isn't that good)

[Enter steps to reproduce below:]

1. Close all files/panes.
2. Press Ctrl+AltGr+\

**Atom Version**: 1.0.7
**System**: *** (corrected. Windows 7)
**Thrown From**: [space-tab](https://github.com/jamesmacfie/space-tab) package, v0.4.2


### Stack Trace

Uncaught TypeError: Cannot read property 'getTabLength' of undefined

```
At C:\Users\%username%\.atom\packages\space-tab\lib\space-tab.coffee:27

TypeError: Cannot read property 'getTabLength' of undefined
  at Object.module.exports.convertToSpaces (C:\Users\%username%\.atom\packages\space-tab\lib\space-tab.coffee:27:22)
  at atom-workspace.<anonymous> (C:\Users\%username%\.atom\packages\space-tab\lib\space-tab.coffee:4:74)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (C:\Users\%username%\AppData\Local\atom\app-1.0.7\resources\app.asar\src\command-registry.js:241:29)
  at CommandRegistry.handleCommandEvent (C:\Users\%username%\AppData\Local\atom\app-1.0.7\resources\app.asar\src\command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (C:\Users\%username%\AppData\Local\atom\app-1.0.7\resources\app.asar\src\command-registry.js:156:19)
  at EventEmitter.<anonymous> (C:\Users\%username%\AppData\Local\atom\app-1.0.7\resources\app.asar\src\window-event-handler.js:70:30)
  at emitOne (events.js:77:13)
  at EventEmitter.emit (events.js:166:7)

```

### Commands

```
     -0:45.5.0 editor-stats:toggle (atom-workspace.workspace.scrollbars-visible-always)
     -0:19.6.0 pane:split-right (ul.list-inline.tab-bar.inset-panel)
     -0:17.7.0 pane:split-left (atom-pane.pane.active)
     -0:08.2.0 space-tab:convert-to-spaces (atom-pane.pane.active)
```

### Config

```json
{
  "core": {
    "projectHome": "D:\\Users\\%username%\\GitHub",
    "disabledPackages": [
      "metrics",
      "autocomplete-emojis",
      "atom-php-navigation",
      "highlight-column"
    ],
    "themes": [
      "atom-dark-ui",
      "one-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
aligner-php, v1.0.0
autocomplete-php, v0.3.7
autohide-tree-view, v0.24.3
color-picker, v2.0.11
editor-stats, v0.17.0
file-icons, v1.6.7
highlight-selected, v0.10.1
keyboard-localization, v1.4.7
language-autoit, v0.2.1
language-batch, v0.3.3
language-ini, v1.11.0
linter, v1.4.3
minimap, v4.13.1
minimap-bookmarks, v0.1.0
minimap-find-and-replace, v4.3.0
minimap-git-diff, v4.1.8
minimap-highlight-selected, v4.3.0
minimap-pigments, v0.1.6
minimap-selection, v4.3.0
php-analyser, v0.2.0
pigments, v0.11.0
space-tab, v0.4.2
travis-ci-status, v0.16.0

# Dev
No dev packages
```
